util/json.go

```diff
@@ -13,9 +13,9 @@ func ReadJSONInto(r io.ReadCloser, data interface{}) error {
 	defer r.Close()
 	bytes, err := ioutil.ReadAll(r)
 	if err != nil {
-		return errors.Wrap(err, "error reading JSON")
+		return errors.Wrapf(err, "error reading JSON (%s)", string(bytes))
 	}
-	return errors.Wrapf(json.Unmarshal(bytes, data), "error attempting to unmarshal into %T: %s", data, string(bytes[:]))
+	return errors.Wrapf(json.Unmarshal(bytes, data), "error attempting to unmarshal into %T: %s", data, string(bytes))
```

Comment:
do we worry about leaking data in an awkward way?

Reply:
fair, removed